### DISCUSSION TITLE
How to handle rest time between sets or exercises?

### DISCUSSION THREAD

**Discussion by adrianlzt** [wger-project/wger#1566] [Q&A]
Is a feature to be added or I?m missing some option to configure rest time between sets and between exercises?

Thanks!

**Comment by rolandgeider** [comment]
Hey! At the moment this can't be configured, in the gym mode there is just a timer that counts up so it's up to you to check how long the pauses should be (but I hope this will be done along with the more flexible workouts)

**Reply by rolandgeider** [reply]
NB: this is configurable (even with progression rules) in the flexible routines branch which should be merged soon
